openreview/tools.py

```diff
@@ -166,68 +161,6 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
     else:
         raise openreview.OpenReviewException('There is already a profile with this email address: {}'.format(email))
 
-
-def create_authorid_profiles(client, note, print=print):
-    # for all submissions get authorids, if in form of email address, try to find associated profile
-    # if profile doesn't exist, create one
-    created_profiles = []
-
-    def clean_name(name):
-        '''
-        Replaces invalid characters with equivalent valid ones.
-        '''
-
-        return name.replace('’', "'")
-
-    def get_names(author_name):
-        '''
-        Splits a string into first and last (and middle, if applicable) names.
-        '''
-
-        names = author_name.split(' ')
-        if len(names) > 1:
-            first = names[0]
-            last = names[-1]
-            middle = ' '.join(names[1:-1])
-            return [clean_name(n) for n in [first, last, middle]]
-        else:
-            return []
-
-    if 'authorids' in note.content and 'authors' in note.content:
-        author_names = [a.replace('*', '') for a in note.content['authors']]
-        author_emails = [e for e in note.content['authorids']]
-        if len(author_names) == len(author_emails):
-
-            email_by_authorname = match_authors_to_emails(author_names, author_emails)
```

Comment:
can you remove this call and keep this function?

I think we should keep it. This function create profiles for current submissions.

Reply:
`get_names` has been restored.